streams/piping/multiple-propagation.js

```diff
@@ -35,6 +35,32 @@ promise_test(t => {
     ]);
   });
 
+}, 'Piping from an errored readable stream to an erroring writable stream');
+
+promise_test(t => {
+  const rs = recordingReadableStream({
+    start(c) {
+      c.error(error1);
+    }
+  });
+  const ws = recordingWritableStream({
+    start(c) {
+      c.error(error2);
+    }
+  });
+
+  // Trying to abort a stream that was errored will give that error back
+  return Promise.resolve()
```

Comment:
The test might be more obviously correct if you checked the errored state of `ws` before doing the piping, instead of checking it at the end.

Reply:
Done. I also split a couple more tests into erroring / errored variants, although in these cases the asserts don't change. PTAL.